### PR TITLE
add attribute support to writeAvros and writeProtos

### PR DESCRIPTION
**Pull request by reuvenlax**

### CODE REVIEW

**Comment by reuvenlax** [comment]
R: @slilichenko

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java [hunk 1]
```diff
@@ -731,8 +731,9 @@ public static Write<PubsubMessage> writeMessagesDynamic() {
    */
   public static Write<String> writeStrings() {
     return Write.newBuilder(
-            (String string) ->
-                new PubsubMessage(string.getBytes(StandardCharsets.UTF_8), ImmutableMap.of()))
+            (ValueInSingleWindow<String> string) ->
```
Comment (slilichenko) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java:734]: Perhaps change the parameter name from "string" to something like "stringAndWindow" or something better.

**Review by slilichenko — COMMENTED** [review]
Minor comment

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java (hunk 1 above)
Comment (reuvenlax) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java]: done

**Review by slilichenko — APPROVED** [review]
LGTM

**Pull request merged**